src/Sets/SparsePolynomialZonotope.jl

```diff
@@ -0,0 +1,296 @@
+export SparsePolynomialZonotope, expmat, nparams, ndependentgens, nindependentgens, ngens,
+        dependent_genmat, independent_genmat, indexvector,
+       linear_map, quadratic_map, remove_redundant_generators
+
+"""
+    SparsePolynomialZonotope{N, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}, MNI<:AbstractVecOrMat{N}, ME<:AbstractMatrix{<:Integer}, VI<:AbstractVector{<:Integer}} <: AbstractPolynomialZonotope{N}
+
+Type that represents a sparse polynomial zonotope.
+
+A sparse polynomial zonotope ``\\mathcal{PZ} ⊂ \\mathbb{R}^n`` is represented by the set
+```math
+\\mathcal{PZ} = \\left\\{x \\in \\mathbb{R}^n : x = c + ∑ᵢ₌₁ʰ\\left(∏ₖ₌₁ᵖ α_k^{E_{k, i}} \\right)Gᵢ+∑ⱼ₌₁^qβⱼGIⱼ,~~ α_k ∈ [-1, 1]~~ ∀ i = 1,…,p, j=1,…,q \\right\\},
+```
+where ``c ∈ \\mathbb{R}^n`` is the offset vector (or center), ``G ∈ \\mathbb{R}^{n \\times h}`` is the dependent generator matrix with columns ``Gᵢ``,
+``GI ∈ \\mathbb{R}^{n×q}`` is the independent generator matrix and where ``E ∈ \\mathbb{N}^{p×h}_{≥0}`` is the exponent matrix with matrix elements ``E_{k, i}``.
+
+### Fields
+
+- `c` -- offset vector
+- `G` -- dependent generator matrix
+- `GI` -- independent generator matrix
+- `E` -- exponent matrix
+- `idx` -- identifiers vector
```

Comment:
```suggestion
- `c`   -- offset vector
- `G`   -- dependent generator matrix
- `GI`  -- independent generator matrix
- `E`   -- exponent matrix
- `idx` -- identifier vector
```

Reply:
The identifier vector is not explained in the text. You can copy the description from the getter.